### PR TITLE
Add stackblitz links and info to issue template

### LINKED ISSUE #56: Create a Stackblitz/issue repro method
Enable devs to report bugs/issues easily by providing a Stackblitz to reproduce them

### PR DESCRIPTION
**Pull request by frehner**
### Description

Closes #56

### Additional context

---

### Before submitting the PR, please make sure you do the following:

- [x] Read the [Contributing Guidelines](https://github.com/shopify/hydrogen-ui/blob/main/contributing.md)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`)
- [ ] Update docs in this repository according to your change
- [ ] Run `yarn changeset add` if this PR cause a version bump based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/). If you have a breaking change, it will need to wait until the next major version release. Otherwise, use patch updates even for new features. Read [more about Hydrogen-UI's versioning.](https://github.com/shopify/hydrogen-ui/blob/main/readme.md#versioning)

### CODE REVIEW

File: apps/nextjs/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)
+
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (cartogram) [inline apps/nextjs/README.md:3]: Should we update the text of the rest of this readme?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
+
+Please use our NextJS issue template here: [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)
```
Comment (cartogram) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
Please use our NextJS stackblitz template to create a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Get started by clicking the button below:
[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)
```

**Review by cartogram — APPROVED** [review]

File: apps/nextjs/README.md (hunk 1 above)
Comment (cartogram) [reply apps/nextjs/README.md:3]: I'm not sure. I think leave it for now and maybe we revisit when there are multiple apps. @gfscott do you have any opinion on the content of this readme?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -10,7 +10,11 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
+
```
Comment (elisenyang) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: What do you think about switching the order around like this?

**To Reproduce**
Steps to reproduce the behavior:

Go to '...'
Click on '....'
Scroll down to '....'
See error

If you're using Hydrogen UI with NextJS, you can use our template to create a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Get started by clicking the button below:
[Open in StackBlitz](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)

**Review by elisenyang — CHANGES_REQUESTED** [review]

**Pull request merged**